portfolio/src/main/webapp/copyright.js

```diff
@@ -1,2 +1,2 @@
 // This displays the current year of the copyright.
-document.getElementById("year").innerHTML = new Date().getFullYear();
\ No newline at end of file
+document.getElementById("copyright-year").innerHTML = new Date().getFullYear();
```

Comment:
I just realized this uses .innerHTML. Are you sure this is necessary? If not, I recommed .innerText instead.

Reply:
I keep forgetting not to use innerHTML. I will make the change as soon as possible and send a PR on this branch.

Reply:
I have made the correction. I am done with my portfolio!